doc/source/examples/incompressible-flow/2d-flow-around-cylinder/2d-flow-around-cylinder.rst

```diff
@@ -0,0 +1,95 @@
+======================================
+2D Flow around a cylinder
+======================================
+
+In this example, a flow is passing across a fixed cylinder. The velocity profile of the flow is simulated. The parameter file used is ``cylinder.prm``. This example can be solved using the ``gls_navier_stokes_2d`` solver.
+
```

Comment:
Can you add the same subsections as in the other examples?
(Features, location of the example, etc.)
:)